<!DOCTYPE html>
<html>

<head>
    <meta charset="UTF-8">
    <link rel="stylesheet" href="bootstrap/css/bootstrap.min.css">
    <link rel="stylesheet" type="text/css" href="css/swiper-bundle.min.css" />
    <link rel="stylesheet" type="text/css" href="css/main.css" />
    <link rel="stylesheet" type="text/css" href="css/team.css" />
    <link rel="stylesheet" type="text/css" href="css/muman1.css" />
    <script src="bootstrap/js/jquery.min.js"></script>
    <script src="bootstrap/js/bootstrap.min.js"></script>
    <script type="text/javascript" src="js/swiper-bundle.min.js"></script>
    <script type="text/javascript" src="js/main.js"></script>
    <script type="text/javascript" src="js/muban1.js"></script>
    <style>
        .left_nav {
            text-align: left;
        }
    </style>
    <title>ED-NAU Results</title>
    <link rel="icon" href="./images/2021IDEClogo设计.png" type="image/x-icon">
</head>

<body>
    <div class="nav col-lg-12 col-md-12 col-sm-12 col-xs-12">
        <div class="logo_container col-lg-3 col-md-3 col-sm-3 col-xs-3">
            <a href="./">
                <img class="homepage_logo" src="./images/2021IDEClogo设计.png" alt="">
            </a>
        </div>
        <div class="nav_cells col-lg-9 col-md-9 hidden-sm hidden-xs">
            <ul>
                <li class='nav_one hover_underline'>
                    <a href="./">HOME</a>
                </li>
                <li class='nav_one hover_underline'>
                    <a href="project-description">PROJECT</a>
                    <ul class="nav_two">
                        <li>
                            <a href="project-description" class="hover_underline">Description</a>
                        </li>
                        <li>
                            <a href="project-design" class="hover_underline">Design</a>
                        </li>
                        <li>
                            <a href="project-protocol" class="hover_underline">Protocol</a>
                        </li>
                    </ul>
                </li>
                <li class='nav_one hover_underline'>
                    <a href="results">RESULTS</a>
                </li>

                <li class='nav_one hover_underline'>
                    <a href="team-members">TEAM </a>
                    <ul class="nav_two">
                        <li>
                            <a href="team-members" class="hover_underline">Members</a>
                        </li>
                        <li>
                            <a href="team-collaborations" class="hover_underline">Collaborations</a>
                        </li>
                        <li>
                            <a href="team-attributions" class="hover_underline">Attributions</a>
                        </li>
                        <li>
                            <a href="team-notebook" class="hover_underline">Notebook</a>
                        </li>
                    </ul>
                </li>
                <li class='nav_one hover_underline'>
                    <a href="lab-notebook">LAB NOTEBOOK</a>
                </li>
            </ul>
        </div>
        <div class="nav_hidden nav_cells hidden-lg hidden-md col-sm-9 col-xs-9">
            <ul>
                <li class="nav_one"><img src="images/nav_little.png" alt="" style='width:7vw'>
                    <ul class="nav_two">
                        <li class="hover_underline">
                            <a href="./">HOME</a>
                        </li>
                        <li class="hover_underline">
                            <a href="project-description">PROJECT</a>
                        </li>
                        <li class="hover_underline">
                            <a href="results">RESULTS</a>
                        </li>
                        <li class="hover_underline">
                            <a href="team-members">TEAM</a>
                        </li>
                        <li class="hover_underline">
                            <a href="lab-notebook">LAB NOTEBOOK</a>
                        </li>
                    </ul>
                </li>
            </ul>
        </div>
    </div>
    <!-- body -->
    <div id="teambody">
        <div class="swiper-container">
            <div class="swiper-wrapper">
                <div class="swiper-slide" style="background-image: url(images/result.JPG);display: flex;justify-content: center;align-items: center;">
                    <img src="./images/result-字.png" alt="" style="flex:1;">
                </div>
            </div>
            <!-- Add Arrows -->
            <div class="swiper-scrollbar"></div>
        </div>
        <div class="left_nav_container col-lg-3 col-md-3 hidden-sm hidden-xs">
            <div class="left_nav">
                <div class="left_nav_img"></div>
                <a href="#ConstructionofthemutantlibraryofOsNramp5">Construction of the mutant library of
                    OsNramp5</a>
                    <a href='#Screeningoferror-pronePCR'>Screening of error-prone PCR random mutation library</a>
                <a href="#Cadmiumrespondingbiosensor">Cadmium responding biosensor</a>
                <a href="#Bioinformatics-assistedmutationsitesselection">Bioinformatics-assisted mutation sites
                    selection</a>
                <a href="#Reference">Reference</a>
            </div>
        </div>
        <div class="col-lg-9 col-md-9 col-sm-12 col-xs-12" style="background-color: white;padding: 4vh 0;">
            <div class="col-lg-1 col-md-1 hidden-sm hidden-xs"></div>
            <div class="col-lg-8 col-md-8 col-sm-12 col-xs-12 content">
                <h1 style="margin:4vh 0;">Overview</h1>
                <p style="font-size: 1.5em;"><b> Construction of the mutant library of OsNramp5</b></p>
                <p><b>Establishment of error-prone PCR experimental system</b></p>
                <p><b>Construction of the random mutation library with homologous recombination</b></p>
                <p style="font-size: 1.5em;"><b>Screening of error-prone PCR random mutation library</b></p>
                <p><b>Determination of Cd<sup>2+</sup>concentration in solid media</b></p>
                <p><b>Screening of OsNramp5-mut strains transferring less Cd<sup>2+</sup></b></p>
                <p><b>Screening of <i>OsNramp5-mut</i>strains transporting Mn<sup>2+</sup> normally</b></p>
                <p style="font-size: 1.5em;"><b>Cadmium responding biosensor</b></p>
                <p style="font-size: 1.5em;"><b>Bioinformatics-assisted mutation sites selection</b></p>
                <p><b>Prediction of OsNramp5 protein topological structure</b></p>
                <p><b>Prediction of OsNramp5 protein mutation sites</b></p>
                <p style="font-size: 1.5em;"> <b>Reference</b></p>

                <a id='ConstructionofthemutantlibraryofOsNramp5' class="target-fix"></a>
                <h2>Construction of the mutant library of OsNramp5 </h2>
                <p><b>Establishment of error-prone PCR experimental system. </b></p>
                <p>Recombinant plasmid pFL61-OsNramp5 was constructed by homologous recombination. The mutation
                    frequency of the library was positively correlated with the amount of StarMut Enhancer added into
                    the reaction system. The reaction system contained
                    0, 1, 2.5, 5, 10, 20 μL StarMut Enhancer to obtain PCR amplification of OsNramp5 cDNA sequence.
                    According to the result of electrophoresis, 10 μL StarMut Enhancer performed the best result. The
                    mutant ratio is about 6.5‰ when 10 μL
                    StarMut Enhancer is added into the reaction system, according to kit instructions. The optimum
                    temperature of error-prone PCR is 63.2 ℃ according to the gradient experiment. </p>
                <div class="content_img">
                    <img src="./images/result1.jpg" alt="" class="result_viewer">
                    <p><b>Fig.1 Error-prone PCR results at gradient temperature. </b> Minimal dispersion can be observed
                        at 63.2 °C, while the brightness of the bands indicated a more successful amplification.
                        Therefore, the annealing temperature was set
                        as 63.2 ℃ in the subsequent experiments. </p>
                </div>
                <p><b>Construction of the random mutation library with homologous recombination. </b></p>
                <p>Linearized pFL61 plasmids and OsNramp5 mutant sequences were used to conduct homologous recombination
                    and built a mutant plasmid library. After the plasmid library was constructed, yeast transformation
                    was performed and screening could
                    be carried out using SD-Ura solid media. Thirteen rounds of plasmids were transformed and about 200
                    plates were plated in 6 weeks. After 72 hours of incubation in 30 ℃, positive transformants were
                    re-delineated. Twenty-one library
                    plates (Fig.2) were established with a total capacity of about 260 colonies. During our research, we
                    found that the homologous recombinant plasmids showed instability in heredity and low efficiency in
                    the process of transformation.
                    The plasmids also came out with the problem during PCR. We could observe smears or loss of our
                    target bands. We supposed that the simple homologous recombination process lacks the common
                    methylation and phosphorylation which happen
                    normally in common cells, thus decreasing the stability of the plasmids and leading to their
                    breakage during PCR. </p>
                <p>Afterwards, to solve this problem, we made attempt to transform the mutant recombinant plasmids into
                    <i>E.coli (Escherichia coli) </i> DH5α for amplifying firstly, and then transferred them into
                    <i>Saccharomyces cerevisiae Δycf1</i> .The results revealed that the efficiency of transformation
                    was greatly improved. The plates covering 100μL yeast solution showed up to almost 200 CFUs per
                    plate, far exceeding the previous 2 to
                    30 CFUs. However, after the plasmid amplification in
                    <i>E.coli</i> the number of the same yeast clones turned out to be huge as well. 115 strains were
                    selected randomly to fill in the mutation library.
                </p>
                <div class="content_img">
                    <div style="white-space: nowrap;">
                        <img src="./images/result2.jpg" alt="" class="result_viewer"
                            style="width: auto;height: 20vh;display: inline-block;">
                        <img src="./images/result3.jpg" alt="" class="result_viewer"
                            style="width: auto;height: 20vh;display: inline-block;">
                    </div>
                    <p><b>Fig. 2 Part of the mutant libiaries. </b></p>
                </div>
                <a id='Screeningoferror-pronePCR' class="target-fix"></a>
                <h2>Screening of error-prone PCR random mutation library</h2>
                <p><b>Determination of Cd<sup>2+</sup>concentration in solid media. </b></p>
                <p> Solid SD-Ura media were set up with different Cd<sup>2+</sup> concentration: 0, 30, 40, 50, 100 μM,
                    to find a proper Cd<sup>2+</sup>concentration. We tried to get a concentration at which the
                    phenotype of yeast transferred to unedited
                    pFL61 plasmid as control group was significantly better than that in <i>HR</i>, the yeast containing
                    <i>OsNramp5-ori</i>. The solid media with too much Cd
                    <sup>2+</sup> led to poor phenotype for <i>pFL61</i> and <i>HR</i>, while the low Cd<sup>2+</sup>
                    concentration in the solid media results in good growth for control and they can't make any
                    difference. In the two occasions mentioned
                    above, we can't decide if the phenotype of yeast in experiment group (<i>OsNramp5-mut</i> yeast) is
                    close to that of <i>Saccharomyces cerevisiae</i> that we inserted in <i>pFL61</i> or <i>HR</i>
                    (<i>OsNramp5-ori Saccharomyces cerevisiae</i>).
                    Yeasts with a similar phenotype to <i>pFL61</i> are defined as potential benign mutants while yeasts
                    with phenotypes similar to or worse than <i>HR</i> are considered to contain deleterious mutation
                    sites or null mutation sites. The
                    phenotype difference between <i>pFL61</i> group and <i>HR</i> group is the most evident when the
                    concentration of Cd<sup>2+</sup> is 30 μM. The growth state is suitable for observation after being
                    cultured for 72h. Therefore, the concentration
                    of Cd<sup>2+</sup> in the solid SD-Ura media was set as 30 μM in the following experiment.
                </p>
                <div class="content_img">
                    <img src="./images/result4.jpg" alt="" class="result_viewer">
                    <p><b>Fig.3 The growth of yeast at different Cd<sup>2+</sup> concentrations. </b> The most suitable
                        concentration for screening potential benign mutant strains was 30 μM. </p>
                </div>
                <p><b>Screening of OsNramp5-mut strains transferring less Cd<sup>2+</sup>.</b></p>
                <p>Recombinant plasmids were transformed into yeasts, and they could not grow on the solid SD-Ura media
                    without this kind of plasmid. In the process of library construction, we got 260 positive
                    transformants totally. Then, the single colony
                    of each transformant was propagated in liquid SD-Ura media. Different densities of yeast were
                    obtained by gradient dilution, to spot on SD-Ura solid media containing Cd<sup>2+</sup>. By doing
                    so, the effect of different yeast concentration
                    was further avoided on the final phenotypic observation. 6 mutant strains (Fig.4) were selected by
                    spot experiments, of which the phenotype was close to the control group. Therefore, they were
                    recognized as potential benign mutants.
                    These mutants were further sequenced and analyzed, in order to explore associations among these
                    mutation sites in different mutants. Besides, we also obtained mutants with poorer phenotypes than
                    <i>HR</i>. We concluded that OsNramp5
                    of these mutants probably increased their ability to transport Cd<sup>2+</sup>. Therefore, the
                    mutation sites of these mutants were probably related to Cd<sup>2+</sup> transportation. </p>
                <p>Six mutants with benign phenotype we selected were sequenced, and sequences of two mutants were
                    successfully obtained. Some mutation sites were finally found in OsNramp5 of ep-17: H21L, P22A,
                    Q23P, K26Q, L78Q; in OsNramp5 of ep-11: Q78L,
                    S162G, Q167L, T294A. Due to frame shift mutation, ep5-11 was terminated early at 298<sup>th</sup>
                    site and ep5-17 was terminated early at the 114
                    <sup>th</sup> site. We analyzed that the structure of the protein was damaged resulting in the loss
                    of its ability to transport cadmium, and this change was likely to also affect the transport of Mn
                    <sup>2+</sup>.
                </p>
                <div class="content_img">
                    <div style="white-space: nowrap;">
                        <img src="./images/result5.jpg" alt="" class="result_viewer"
                            style="width: auto;height: 20vh;display: inline-block;">
                        <img src="./images/result6.jpg" alt="" class="result_viewer"
                            style="width: auto;height: 20vh;display: inline-block;">
                    </div>
                    <div style="white-space: nowrap;">
                        <img src="./images/result7.jpg" alt="" class="result_viewer"
                            style="width: auto;height: 20vh;display: inline-block;">
                        <img src="./images/result8.jpg" alt="" class="result_viewer"
                            style="width: auto;height: 20vh;display: inline-block;">
                    </div>
                    <p><b>Fig.4 Potential benign mutant strains ep 5-11 and epⅠ. </b> The growth of ep 5-11 and epⅠ on
                        SD-Ura plates contains Cd<sup>2+</sup> concentration of 30 μM. It was slightly better than that
                        of <i>pFL61</i> and <i>HR</i> while they
                        grew similarly to <i>pFL61</i> and <i>HR</i> on plates without Cd<sup>2+</sup>, indicating that
                        ep 5-11 and epⅠmay be the potential OsNramp5-mut strains. Its ability of transporting
                        Cd<sup>2+</sup> has successfully decreased or
                        their OsNramp5 has been damaged. </p>
                </div>
                <p><b>Screening of <i>OsNramp5-mut</i> strains transporting Mn<sup>2+</sup> normally. </b></p>
                <p>The mutants with reduced transport of Cd<sup>2+</sup> selected in primary screening were treated as
                    new experimental materials. They were spotted on Mn<sup>2+</sup>-deficient SD-Ura media with the
                    control group (<i>Δsmf1-pFL61</i> and
                    <i>Δsmf1-HR</i>). The Mn<sup>2+</sup> in media was specifically chelated by EGTA. As for the mutant
                    strains, <i>Δsmf1-pFL61</i> could not grow well in the media while <i>Δsmf1-HR</i> can, because the
                    <i>Δsmf1</i> strain cannot transport
                    Mn
                    <sup>2+</sup> normally, they already had an additional Mn<sup>2+</sup> OsNramp5 transporter whose
                    ability of transporting Cd<sup>2+</sup> is decreased. If its ability to transport Mn<sup>2+</sup> is
                    not affected by the mutation, its
                    growth phenotype should be similar to <i>HR</i>. If the function of OsNramp5 in transporting
                    Mn<sup>2+</sup>is damaged during the mutation, its growth phenotype should be similar to that of
                    <i>pFL61</i>.
                </p>
                <p>The re-screening strategy using Mn<sup>2+</sup> was exactly the same as the primary screening by Cd
                    <sup>2+</sup>. Series of densities of yeast solution was set for spot experiment. In conclusion, the
                    strategy to select the benign OsNramp5-mut which can transport Mn<sup>2+</sup> normally is that the
                    closer the phenotype of the mutant
                    is to <i>HR</i>, the more likely it is to be the benign mutant strain we expect. Due to the time
                    limitation, it was difficult for us to obtain this part of the data before the paper was submitted,
                    but relevant experiments have been
                    conducted. If the experiments go smoothly, we hope to be allowed to present our final screening
                    results in the thesis defense.
                </p>
                <a id='Cadmiumrespondingbiosensor' class="target-fix"></a>
                <h2>Cadmium responding biosensor </h2>
                <div class="content_img">
                    <img src="./images/result9.jpg" alt="" class="result_viewer">
                    <p><b>Fig.5 Effect of Cd<sup>2+</sup> concentration on engineered bacteria. </b></p>
                </div>
                <p>Researchers have found that some microorganisms show good ability in Cd<sup>2+</sup> resistance and
                    removal (Albano & Macfie, 2016). Especially, <i>Pseudomonas aeruginosa</i> is a heavy-metal
                    resistant bacteria isolated from industry wastewater,
                    which possesses high resistance to Cd
                    <sup>2+</sup>. Further researches showed that the Cd<sup>2+</sup> resistance ability relies on two
                    metal-recognizing regulatory proteins—P-type ATPase (CadA) and transcription regulatory protein
                    (CadR). The CadR protein has high binding
                    affinity with the promoter of <i>cadR</i> and <i>cadA</i> (P
                    <sub><i>cadR</i></sub> and P<sub><i>cadA</i></sub>) and represses the expression of the CadA genes
                    downstream. However, in the presence of Cd<sup>2+</sup>, CadR will binding to Cd<sup>2+</sup> with
                    high specificity, thus decreases
                    its affinity to its promoter. So this protein acts as a strong repressor of transcription
                    corresponding to the change of Cd<sup>2+</sup> concentration.
                </p>
                <p>In our project, we constructed a gene circuit with <i>cadR</i>, P<sub><i>cadA</i></sub> and green
                    fluorescent protein (GFP) in <i>Escherichia coli</i>. The GFP fluorescence intensity may change
                    according to the Cd<sup>2+</sup> concentration,
                    the higher the Cd<sup>2+</sup> concentration is, the stronger the fluorescence intensity may be,
                    hoping that it can act as a microbial biosensor to present the intracellular cadmium concentration
                    which affected by different OsNramp5.
                </p>
                <p>The characterization experiment showed that, the <i>E.coil</i> BL21 which contained recombinant
                    plasmids got higher fluorescence intensity compared with non-loaded BL21 and BL21 which transformed
                    plasmids with GFP only. At around 2 hours,
                    the fluorescence intensity of engineered bacteria in 10 μM Cd<sup>2+</sup> solution was higher than
                    that in 1 μM Cd<sup>2+</sup> solution, since the bacteria needed some time to adapt to the
                    environment of Cd<sup>2+</sup>. But in general,
                    there was a positive correlation between Cd<sup>2+</sup> concentration and GFP fluorescence
                    intensity. However, the engineered bacteria was less responsive to 100 μM Cd<sup>2+</sup>
                    comparatively. It was probably because the engineered
                    bacteria exhibited lower resistance to cadmium when the concentration is too high, since the
                    physicochemical properties of the bacteria changed or part of the GFP protein denatured. </p>
                <a id='Bioinformatics-assistedmutationsitesselection' class="target-fix"></a>
                <h2>Bioinformatics-assisted mutation sites selection </h2>
                <p>Directed evolution, by establishing mutation libraries and using high-throughput screening methods,
                    is a common strategy for protein design and modification in protein engineering, which can rapidly
                    improve the specific properties of the
                    protein. Directed evolution brings infinite possibilities for protein modification. Discontinuous
                    directed evolution relying on traditional methods often results in huge mutation libraries,
                    consuming a lot of time and energy, ending
                    up with bad results obtained often
                    <sup>[1]</sup>. In recent years, with the improvement of computer power and the optimization of
                    algorithms, the computer-assisted directed evolution for protein has developed quickly, which has
                    opened up a new direction for directed
                    evolution. Protein computational design based on structural simulation and energy calculation
                    assisting to select mutation site provides insights into "precise" directed evolution<sup>[2]</sup>.
                    Therefore, bioinformatics analysis can
                    help to reduce the redundant work for the construction of discontinuous directed evolutionary
                    mutation scheme. In this study, the error-prone PCR strategy was adopted to directionally evolve
                    OsNramp5. Due to its large error-prone mutation
                    library and unconcentrated mutation sites, the ideal result was not obtained. We intend to use this
                    method to analyze our target protein and provide theoretical reference for other researchers.
                </p>
                <a id='PredictionofOsNramp5proteintopologicalstructure' class="target-fix"></a>
                <p><b>Prediction of OsNramp5 protein topological structure </b></p>
                <p>OsNramp5 is a typical membrane protein<sup>[3]</sup>. The topological structure of a membrane protein
                    can be defined as its transmembrane way, the position of the membrane segment in the amino acid
                    sequence and the overall orientation
                    of the protein molecule on the membrane. The secondary structure of membrane protein can be
                    represented by its topological structure, which can be regarded as the intermediate of membrane
                    protein folding, and is the starting point
                    for predicting the tertiary structure of membrane protein. In order to learn more about OsNramp5
                    protein structure, its protein topology structure is predicted through TMHMM v2.0 (<a
                        href="http//www.cbs.dtu.dk/services/TMHMM/">http//www.cbs.dtu.dk/services/TMHMM/</a>).
                    And it turns out that OsNramp5 has 12 typical transmembrane helical regions and 76.3% of the protein
                    terminal are in the membrane, which is structurally similar to other Nramp families
                    <sup>[4]</sup>(Table S2).
                </p>
                <a id='PredictionofOsNramp5proteinmutationsites' class="target-fix"></a>
                <p><b>Prediction of OsNramp5 protein mutation sites </b></p>
                <p>Mutation hotspots of the OsNramp5 have been analyzed by searching related papers and comparing with
                    structure of channel proteins from the same family that have been analyzed. The sites 60D, 63N,
                    157G, 180E and 235M were thought to be
                    important for absorbing Mn2+ in OsNramp5. Different mutations of 235M result in different states of
                    absorbing Mn2+ and Cd2+<sup>[5,6]</sup>. The sites 60D and 63N are located outside membrane domain;
                    the sites 157G, 180E and 235M are
                    respectively located in S4, S5 and S6 transmembrane domain. "DPGN" located in the first
                    transmembrane domain is a conserved domain (Table S3), and is thought to be an important domain
                    interacting with Mn2+ homologous modeling for OsNramp5
                    by SWISS-MODEL<sup>[5,7-10]</sup>. The model is based on the transmembrane protein of Eremococcus
                    coleocola MntH (PDB Code: 6TL2.1). This figure is made by PyMOL (Fig.7). These potential mutation
                    sites are all from the previous reports
                    or software prediction. Accuracy and specific effects of these sites in OsNramp5 have not been
                    tested, and only taken as reference.
                </p>
                <div class="content_img">
                    <img src="./images/result10.jpg" alt="" class="result_viewer">
                    <p><b>Fig.7 OsNramp5 metal transporter.</b>The highly conserved "DPNG" region, located at the
                        c-terminus of the first transmembrane region, is consistent with the calibration of 12 cadmium
                        transmembrane residues in Nramp family homologue
                        by Gros et al. OsNramp5 homologous model was based on Eremococcus coleocola MntH (PDB Code:
                        6TL2.1) transmembrane protein, and its potential metal ion action region was labeled, which was
                        used as the preferred mutation site.</p>
                </div>
                <a id='Reference' class="target-fix"></a>
                <h2>Reference </h2>
                <p>[1] SHELDON R A, PEREIRA P C. Biocatalysis engineering: the big picture [J]. Chemical Society
                    Reviews, 2017, 46(10): 2678-91.</p>
                <p>[2] YANG K K, WU Z, ARNOLD F H. Machine-learning-guided directed evolution for protein engineering
                    [J]. Nature Methods, 2019, 16(8): 687-94.</p>
                <p>[3] MANI A, SANKARANARAYANAN K. In Silico Analysis of Natural Resistance-Associated Macrophage
                    Protein (NRAMP) Family of Transporters in Rice [J]. The Protein Journal, 2018, 37(3): 237-47.</p>
                <p>[4] BOZZI A T, GAUDET R. Molecular Mechanism of Nramp-Family Transition Metal Transport [J]. J Mol
                    Biol, 2021, 433(16): 31.</p>
                <p> [5] POTTIER M, OOMEN R, PICCO C, et al. Identification of mutations allowing Natural Resistance
                    Associated Macrophage Proteins (NRAMP) to discriminate against cadmium [J]. Plant Journal, 2015,
                    83(4): 625-37.</p>
                <p>[6] JIYU L. Structure and function of Arabidopsis thaliana metal transporter NRAMPs [D]; Nanjing
                    Agricultural University, 2018.</p>
                <p>[7] BIENERT S, WATERHOUSE A, DE BEER TJAART A P, et al. The SWISS-MODEL Repository—new features and
                    functionality [J]. Nucleic Acids Research, 2016, 45(D1): D313-D9.</p>
                <p>[8] BERTONI M, KIEFER F, BIASINI M, et al. Modeling protein quaternary structure of homo- and
                    hetero-oligomers beyond binary interactions by homology [J]. Scientific Reports, 2017, 7(1): 10480.
                </p>
                <p>[9] STUDER G, REMPFER C, WATERHOUSE A M, et al. QMEANDisCo—distance constraints applied on model
                    quality estimation [J]. Bioinformatics, 2019, 36(6): 1765-71.</p>
                <p>[10] STUDER G, TAURIELLO G, BIENERT S, et al. ProMod3—A versatile homology modelling toolbox [J].
                    PLOS Computational Biology, 2021, 17(1): e1008667.</p>
                <a id="download"></a>
                <br>
                <br>
                <p>Download:</p>

                <a href="Supplementary Information-1.zip" download>Supplementary Information-1</a><br>
                <a href="Supplementary Information-2.rar" download>Supplementary Information-2</a>
            </div>
            <div id="ShowImage_Form" class="modal">
                <div class="modal-body">
                    <div id="img_show">
                    </div>
                </div>
            </div>
        </div>
    </div>
    <div class="footer">
        <div class="footer_img col-lg-3 hidden-md hidden-sm hidden-xs"></div>
        <div class="footer_menu col-lg-8 col-md-12 col-sm-12 col-xs-12">
            <div class="footer_list ">
                <div class="footer_title ">
                    <a href="./ ">
                        <div class="footer_arrow ">—</div>
                        <div>&nbsp;HOME</div>
                    </a>
                </div>
            </div>
            <div class="footer_list ">
                <div class="footer_title ">
                    <a href="project-description ">
                        <div class="footer_arrow ">—</div>
                        <div>&nbsp;PROJECT</div>
                    </a>
                </div>
                <div class="footer_title ">
                    <a href="project-description ">
                        <div class="footer_arrow ">—</div>
                        <div>&nbsp;Description</div>
                    </a>
                </div>
                <div class="footer_title ">
                    <a href="project-design ">
                        <div class="footer_arrow ">—</div>
                        <div>&nbsp;Design</div>
                    </a>
                </div>
                <div class="footer_title ">
                    <a href="project-PROTOCOL ">
                        <div class="footer_arrow ">—</div>
                        <div>&nbsp;Protocol</div>
                    </a>
                </div>
            </div>
            <div class="footer_list ">
                <div class="footer_title ">
                    <a href="results ">
                        <div class="footer_arrow ">—</div>
                        <div>&nbsp;RESULTS</div>
                    </a>
                </div>
            </div>
            <div class="footer_list ">
                <div class="footer_title ">
                    <a href="team-members ">
                        <div class="footer_arrow ">—</div>
                        <div>&nbsp;TEAM</div>
                    </a>
                </div>
                <div class="footer_title ">
                    <a href="team-members ">
                        <div class="footer_arrow ">—</div>
                        <div>&nbsp;Members</div>
                    </a>
                </div>
                <div class="footer_title ">
                    <a href="team-collaborations ">
                        <div class="footer_arrow ">—</div>
                        <div>&nbsp;Collaborations</div>
                    </a>
                </div>
                <div class="footer_title ">
                    <a href="team-attributions ">
                        <div class="footer_arrow ">—</div>
                        <div>&nbsp;Attributions</div>
                    </a>
                </div>
                <div class="footer_title ">
                    <a href="team-notebook ">
                        <div class="footer_arrow ">—</div>
                        <div>&nbsp;Notebook</div>
                    </a>
                </div>
            </div>
            <div class="footer_list ">
                <div class="footer_title ">
                    <a href="lab-notebook">
                        <div class="footer_arrow ">—</div>
                        <div>&nbsp;LAB NOTEBOOK</div>
                    </a>
                </div>
            </div>
        </div>
    </div>
    <script type="text/javascript" src="js/team.js"></script>
</body>

</html>